<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of Points:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="NumOfPoints"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_1"/>
       </item>
       <item>
        <widget class="QwtPlot" name="QwtPlot_1">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_3"/>
       </item>
       <item>
        <widget class="QwtPlot" name="QwtPlot_3">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_5"/>
       </item>
       <item>
        <widget class="QwtPlot" name="QwtPlot_5">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
       <item>
        <widget class="QwtPlot" name="QwtPlot_2">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_4"/>
       </item>
       <item>
        <widget class="QwtPlot" name="QwtPlot_4">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_6"/>
       </item>
       <item>
        <widget class="QwtPlot" name="QwtPlot_6">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>QwtPlot_1</zorder>
  <zorder>QwtPlot_2</zorder>
  <zorder>QwtPlot_3</zorder>
  <zorder>QwtPlot_4</zorder>
  <zorder>QwtPlot_5</zorder>
  <zorder>QwtPlot_6</zorder>
  <zorder>comboBox_1</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>comboBox_4</zorder>
  <zorder>comboBox_5</zorder>
  <zorder>comboBox_6</zorder>
  <zorder>QwtPlot_6</zorder>
  <zorder>NumOfPoints</zorder>
  <zorder>label</zorder>
  <zorder>horizontalSpacer</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
